This patch looks insane but the logic is explained here:

https://trac.macports.org/ticket/61868#comment:46

The 10.4 SDK needs three such pragmas; 10.14 SDK needs five. Add more if you need them; it won't hurt anything.

--- libusb/os/darwin_usb.h.orig
+++ libusb/os/darwin_usb.h
@@ -27,6 +27,9 @@
 
 #include <IOKit/IOTypes.h>
 #include <IOKit/IOCFBundle.h>
+#pragma options align=power
+#pragma options align=power
+#pragma options align=power
+#pragma options align=power
+#pragma options align=power
 #include <IOKit/usb/IOUSBLib.h>
 #include <IOKit/IOCFPlugIn.h>
 
